src/main/java/teammates/ui/request/NotificationBasicRequest.java

```diff
@@ -4,9 +4,9 @@
 import teammates.common.datatransfer.NotificationType;
 
 /**
- * The request for creating a notification.
+ * The basic request for a notification.
  */
-public class CreateNotificationRequest extends BasicRequest {
+public class NotificationBasicRequest extends BasicRequest {
     private Long startTimestamp;
     private Long endTimestamp;
```

Comment:
Should these be `long` instead

Reply:
Fixed in the latest commit. Must have changed during the rebase